packages/about-you/api-client/src/index.ts

```diff
@@ -6,7 +6,7 @@ let apiClient: BapiClient | null = null;
 
 const { setup, update, getSettings } = apiClientFactory<any, any>({
   defaultSettings: {},
-  onSetup: (setupConfig: SetupConfig) => {
+  onCreate: (setupConfig: SetupConfig) => {
```

Comment:
ayc is not aligned with this version anyway so I'd leave it on lower one and don't change anything